clusterloader2/pkg/measurement/common/wait_for_rc_pods.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+func init() {
+	measurement.Register("WaitForRunningRCs", createWaitForRunningRCsMeasurement)
+}
+
+func createWaitForRunningRCsMeasurement() measurement.Measurement {
+	return &waitForRunningRCsMeasurement{}
+}
+
+type waitForRunningRCsMeasurement struct{}
+
+// Execute waits until all specified RCs have all pods running or until timeout happens.
+// RCs can be specified by field and/or label selectors.
+// If namespace is not passed by parameter, all-namespace scope is assumed.
+func (*waitForRunningRCsMeasurement) Execute(config *measurement.MeasurementConfig) ([]measurement.Summary, error) {
+	var summaries []measurement.Summary
+	namespace, err := util.GetStringOrDefault(config.Params, "namespace", metav1.NamespaceAll)
```

Comment:
I don't think you need to have an option for namespace separately. It can be done through field selector on namespace.

Reply:
(Simiarly for wait_for_pods.go)

Reply:
Actually, after thinking again, maybe it's better to keep it as it offers some convenience.